cppcache/integration/test/sni-test-config/geode-config/truststore_sni.pem

```diff
@@ -0,0 +1,68 @@
+-----BEGIN CERTIFICATE-----

```

Comment:
Why do we use a pre-baked key and certificates here? I though all the SSL tests generated certs on the fly but maybe I am wrong. In the past this has caused issues with hostname validation.

Reply:
you are probably thinking of the server side...

Reply:
Is there a technical reason we can’t do it for the client too?

Reply:
technically possible, just a resource constraint